changelogs/unreleased/gh-10479-fix-xlog-open-memleak.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/lua
+
+- Fixed a memory leak on `xlog` open failure (gh-10479).
```

Comment:
Better write: "Fixed a memory leak in `xlog.pairs()` on failure to open the file".

Reply:
Fixed.